### ISSUE TITLE
Society Augmentation Gem quest no longer working since last update.

### ISSUE THREAD

**Issue by VokardIX** [ACEmulator/ACE#3427]
### Bug Report or Feature Request (mark with an `x`)
```
- [ X] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
Go to society (I'm in Hand) and speak with Operations Aid NPC, then talk to Operations Specialist.  They get confirmation from you and portal you into the dungeon.  As you start the quest, there should be two rooms on the West side that are filled with enemies.  Before this last patch there was.  Now it seems there's only one room filled, and it's with the wrong set of enemies.  
The set that's there now, should be part of the next set of rooms.  But there's only one set here now.

### Mention any other details that might be useful.

This was working prior to the current patch, as I run this weekly.

**Comment by LtRipley36706** [comment]
Will be fixed in ACEmulator/ACE-World-16PY-Patches#800

**Comment by VokardIX** [comment]
Awesome!

Thanks for all y'all do.

mv
